src/applications/personalization/dashboard/actions/messaging.js

```diff
@@ -26,6 +28,16 @@ export function fetchFolder(id, query = {}) {
       const folderUrl = `/folders/${id}`;
       const messagesUrl = createUrlWithQuery(`${folderUrl}/messages`, query);
 
+      if (environment.isLocalhost && !window.Cypress) {
+        dispatch({
+          type: FETCH_FOLDER_SUCCESS,
+          folder,
```

Comment:
I was confused by this... I think it'd be a bit clearer if the imports were renamed so that this could read something like: `folder: mockFolderResponse`